BikeSharing360_MC/.gitignore

```diff
@@ -0,0 +1,353 @@
+## Ignore Visual Studio temporary files, build results, and
```

Comment:
Do we need such a verbose gitignore for this?

Reply:
Not really. What is your suggestion? Do we need to remove .gitignore? or just that entry.

Reply:
It's a nit but the .gitignore was just pretty big and it looked like some entries not necessary (not just this one).

Reply:
Got it, yeh it was added from vs/vs code. We haven't added anything extra. So, better would be trim it down and push the changes?

Reply:
If you can, that would be great!

Reply:
ok, one more clarification, we used the existing samples from here https://github.com/microsoft/BikeSharing360_MultiContainer/ where I believe its the same .gitignore content. Is there a way that we can flush it to a default/minmal one? (havent done that before)

Reply:
I don't think there is any easy way. We can just keep this :)